climada/engine/test/test_impact.py

```diff
@@ -69,6 +69,123 @@ def test_from_eih_pass(self):
             np.stack([exp.gdf.latitude.values, exp.gdf.longitude.values], axis=1)
             )
 
+    def test_concat_pass(self):
```

Comment:
Maybe define a new test class instead of a single test? This would make it easier to narrow down the issue if only a single one of the tests is failing